### ISSUE TITLE
Globus linked identities causing confusion

### ISSUE THREAD

**Issue by JoshuaSBrown** [ORNL/DataFed#850]
A single user that has two accounts with Globus and uses both to log into DataFed separately and then links their globus accounts will cause confusion to DataFed, because DataFed will not know they are linked but both accounts will now need to use the same Globus ID.

**Comment by dvstans** [comment]
I don't really know how to address this. Two distinct and un-linked Globus accounts are different users as far as DataFed can see. We could add an admin feature to merge accounts into one, but this would be fairly involved to implement.

**Comment by JoshuaSBrown** [comment]
Yes, this becomes challenging, because what happens to all the aliases if you merge accounts. What if we put the onus on the user to delete one of their accounts and then call a DataFed supported update command that would then be able to update its understanding of the users Globus accounts. I imagine users would identify this problem pretty early on after creating a second account.

**Comment by dvstans** [comment]
I agree. The system registration UI should make it clear that if they create multiple accounts then it's the users problem. We may also be able to look up their user name and warn them if there are close matches already registered.
